### PR TITLE
Add positive feedback to Attorney Evaluation Screen for AMA appeals

### PR DESCRIPTION
**Pull request by jcq**
Connects #7929

### Description
Allow judge to leave positive feedback for attorney in addition to negative

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Attorney feels better about their work upon receiving positive feedback

### Testing Plan
1. Choose BVAAABSHIRE (Judge with hearings & cases)
2. Go to Queue
3. Choose an AMA case
4. Select "Ready for Dispatch"
5. Fill out decision modal
6. Verify that "Identify Positive Feedback" section is functional

### Documentation Updates
- [ ] Topline File Descriptions Added or Updated as Needed

*Schema Changes Only*

* [ ] #appeals-schema notified with summary and link to this PR

### CODE REVIEW

File: client/app/queue/caseEvaluation/JudgeCaseQuality.jsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            hideLabel
+            vertical
+            name={COPY.JUDGE_EVALUATE_DECISION_POSITIVE_FEEDBACK}
+            onChange={(event) => {
+              setPositiveFeedback((prevVals) => ({
+                ...prevVals,
+                [event.target.getAttribute('id')]: event.target.checked
+              }));
+            }}
+            // errorState={highlight && this.qualityIsDeficient(quality) && _.isEmpty(this.state.positive_feedback)}
```
Comment (aroltsch) [inline client/app/queue/caseEvaluation/JudgeCaseQuality.jsx]: Did you mean to leave it commented out?

Comment (jcq) [reply client/app/queue/caseEvaluation/JudgeCaseQuality.jsx]: Nope, this can disappear. Good catch!

**Comment by laurjpeterson** [comment]
Awesome! Here's a screenshot from demo. AAABSHIRE, choosing an AMA case, clicking "Ready for Dispatch" and going through the check out flow. I think the way the text wraps is a little strange looking, so tagging @carodew for a non-urgent look. We can create another ticket to change the styling, as needed.

Screen Shot 2019-09-05 at 9 34 37 AM

**Comment by carodew** [comment]
Yeah the wrapping is a little wonky. I think the text should all left-align as indicated via the pink line here:

positive-feedback-typography

I also wonder if we can create that update in the style guide as well, in case it comes up again?

**Comment by jcq** [comment]
Yeah, good call. I had meant to flag this during development, since it does look a bit wonky. 

We'd have to update the `CheckboxGroup`, which would update all places it is used (including multiple on that page).

**Comment by laurjpeterson** [comment]
ticketed https://github.com/department-of-veterans-affairs/caseflow/issues/11987

**Pull request merged**